### ISSUE TITLE
Site request: moonwalk.cc

### ISSUE THREAD

**Issue by nmr50** [rg3/youtube-dl#8522]
Hello *!
Some months I've used so-called online theaters and noticed that the most of those using moonwalk.cc video source. Yesterday I've tried to view some movie and got "receive code" with this url: http://moonwalk.cc/video/b39bacbfd96c3939/iframe
Y-dl does not know about this source, yet in another 'capture plugin' I found that this source gives chunked output and those small pieces can be played by dedicated video player only.
Can you add this moonwalk.cc site to Y-dl "can do", please:)
Thanks!
-t
Y-dl -v output:
C:\temp\>youtube-dl.exe -v http://moonwalk.cc/video/b39bacbfd96c3939/iframe
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'http://moonwalk.cc/video/b39bacbfd96c3939/iframe']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2016.02.10
[debug] Python version 2.7.10 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-78313-g6632802, rtmpdump 2.4
[debug] Proxy map: {}
[generic] iframe: Requesting header
WARNING: Could not send HEAD request to http://moonwalk.cc/video/b39bacbfd96c3939/iframe: HTTP Error 404: Not Found
[generic] iframe: Downloading webpage
WARNING: Falling back on generic information extractor.
[generic] iframe: Extracting information
ERROR: Unsupported URL: http://moonwalk.cc/video/b39bacbfd96c3939/iframe
Traceback (most recent call last):
  File "youtube_dl\extractor\generic.pyo", line 1294, in _real_extract
  File "youtube_dl\compat.pyo", line 248, in compat_etree_fromstring
  File "youtube_dl\compat.pyo", line 237, in _XML
  File "xml\etree\ElementTree.pyo", line 1642, in feed
  File "xml\etree\ElementTree.pyo", line 1506, in _raiseerror
ParseError: mismatched tag: line 111, column 2
Traceback (most recent call last):
  File "youtube_dl\YoutubeDL.pyo", line 666, in extract_info
  File "youtube_dl\extractor\common.pyo", line 315, in extract
  File "youtube_dl\extractor\generic.pyo", line 1930, in _real_extract
UnsupportedError: Unsupported URL: http://moonwalk.cc/video/b39bacbfd96c3939/iframe

**Comment by dstftw** [comment]
This looks like copyrighted content. youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free). Do they have rights to distribute this? If not provide [an example URL](https://github.com/rg3/youtube-dl/blob/master/README.md#is-the-description-of-the-issue-itself-sufficient) that does not violate any copyrights.

**Comment by nmr50** [comment]
http://moonwalk.cc/video/f6ac8cab63d05c3e/iframe - Teen wolf translated series on moonwalk are freely distributed on MTV network but w/o translation - http://www.mtv.com/full-episodes/2uv24h/teen-wolf-after-after-show-lies-of-ommission-season-5-ep-after-aftr-509)

**Comment by nmr50** [comment]
http://www.cbs.com/shows/limitless/video/oTXWvjiL_DxD_68mNXhfP6qoUl1YX3AG/limitless-the-assassination-of-eddie-morra/ -- CBS free distribution of some Limitless series, yet moonwalk has LF the same but translated ones - http://moonwalk.cc/video/03cdbf37f3990fc1/iframe .

**Comment by dstftw** [comment]
The fact that some series is freely distributed by copyright owner (MTV or CBS in this case) does not mean anybody else is allowed to distrubute it, conversely it's forbidden according to MTV ToS. I did not find any evidence that proves they are allowed to distribute any of these materials. I also did not find any obvious way to upload user content so this site is clearly dedicated to copyright infringement and can't be supported by youtube-dl.
